# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/composite/ContainerLayerComposite.cpp
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/layers/composite/ContainerLayerComposite.cpp
--- a/gfx/layers/composite/ContainerLayerComposite.cpp	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/layers/composite/ContainerLayerComposite.cpp	Wed May 22 22:27:37 2019 +0000
@@ -10,9 +10,9 @@
 #include "Units.h"                           // for LayerRect, LayerPixel, etc
 #include "CompositableHost.h"                // for CompositableHost
 #include "gfxEnv.h"                          // for gfxEnv
-#include "gfxPrefs.h"                        // for gfxPrefs
 #include "mozilla/Assertions.h"              // for MOZ_ASSERT, etc
 #include "mozilla/RefPtr.h"                  // for RefPtr
+#include "mozilla/StaticPrefs.h"             // for StaticPrefs
 #include "mozilla/UniquePtr.h"               // for UniquePtr
 #include "mozilla/gfx/BaseRect.h"            // for BaseRect
 #include "mozilla/gfx/Matrix.h"              // for Matrix4x4
@@ -473,7 +473,7 @@
       layerToRender->RenderLayer(clipRect, geometry);
     }
 
-    if (gfxPrefs::UniformityInfo()) {
+    if (StaticPrefs::UniformityInfo()) {
       PrintUniformityInfo(layer);
     }
 